source/includes/fact-older-version-limitations.rst

```diff
@@ -0,0 +1,42 @@
+- Writes that produce DDL events cannot occur on the source cluster during the 
+  migration. The following events cannot occur: 
+  
+  - ``create``
+  - ``drop``
+  - ``dropDatabase``
+  - ``createIndexes``
+  - ``dropIndexes``
+  - ``collModrefineCollectionShardKey``
+  - ``rename``
+  - ``reshardCollection``
+  - ``shardCollection``
```

Comment:
[Suggestion]

Maybe alphabetize?

Reply:
I'm inclined to leave this as is, or at least with few swaps, because I felt it'd be good to order it this way to users see the most familiar events at the top. If we swapped this to alphabetical, I though it could be a bit jarring for users to first see ``collModRefineCollectionShardKey``

Reply:
Or actually, @ajayvijayakumar123 just to double check is this supposed to be `collModrefineCollectionShardKey` or is this supposed to be `collMod` and `refineCollectionShardKey` separately?

Reply:
yaa you're right lol. i made a lil typo 😅